@types/css-tree.d.ts

```diff
@@ -0,0 +1,21 @@
+declare module "css-tree" {
```

Comment:
why not use @types/css-tree?

Reply:
they are missing types, I also asked the same in dm, time to put a comment and a todo to submit a pr for them later

Reply:
Or get rid of css-tree eventually :)